test/consumeFlow.test.ts

```diff
@@ -142,7 +142,7 @@ describe("Ocean CLI Publishing", function() {
         exec(`npm run cli getDDO ${downloadDatasetDid}`, { cwd: projectRoot }, (error, stdout) => {
             expect(stdout).to.contain(`${downloadDatasetDid}`);
             expect(stdout).to.contain("https://w3id.org/did/v1");
-            expect(stdout).to.contain("Datatoken");
+            // expect(stdout).to.contain("Datatoken");
```

Comment:
I'd suggest we just remove these rather than commenting them out, I don't see us adding them back in